### ISSUE TITLE
Translate fields values

### ISSUE THREAD

**Issue by felipeArnold** [TappNetwork/filament-auditing#33]
There is the possibility of translating fields such as updated to updated on, and database fields such as Name to Nome

<img width="917" alt="Captura de Tela 2024-11-19 às 21 36 04" src="https://github.com/user-attachments/assets/dffac1ef-ef9b-4cd9-860c-40620877b496">

**Comment by Alp3n** [comment]
Hi,
This would be helpful if anyone achieved this. I actually haven't look into it yet but planning.

**Comment by abdullafahem** [comment]
Does this have a solution please we need to translate the events but only the column names are available not the event itself

**Comment by felipeArnold** [comment]
Hello, the events can be changed in the audit configuration file, regarding the columns I created a trait I have a validation for each column of my database where I record in the standard I want

**Comment by abdullafahem** [comment]
Thank you.
But it can change the event also right?

I need to be for example:
"created" => "إنشاء" in arabic language

**Comment by felipeArnold** [comment]
No, for the update and creation events access the config/audit.php file and change the 'events' fields => [

'Created',

'Updated',

'Deleted',

'Restored',

], this will change your events, the one I did earlier is only for the database columns

**Comment by abdullafahem** [comment]
But if I change in the config/audit.php do not work it throws error
